bindata/network/multus/multus.yaml

```diff
@@ -483,14 +483,18 @@ metadata:
     tier: node
     app: whereabouts
 spec:
-  # reconcile loop every 5 minutes, starting at the top of the hour
-  schedule: "*/5 * * * *"
+  # reconcile loop every 15 minutes, starting at the top of the hour
+  schedule: "*/15 * * * *"
+  concurrencyPolicy: Forbid
```

Comment:
Would `Replace` be better here? On one of my test setups, the `ip-reconciler-xxx` pod isn't starting (expected). One pods runs forever trying to start. With `Replace`, it is at least restarted every time period. Not sure which is better.

Reply:
I think I like replace in that case, as well.